python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -991,12 +989,12 @@ def fetch_row_counts(
         # so that the DuckDB lock is released. This is because DuckDB does not allow for
         # opening multiple connections to the same database when a write connection, such
         # as the one dbt uses, is open.
-        block_on_dbt_run = False
```

Comment:
instead of setting this bool + passing to `imap` to handle, we just `list()` the iterable to force the dbt run to complete inline, which is a bit cleaner